src/pages/ItemPools/StatisticsDashboard.jsx

```
@@ -3,7 +3,171 @@ import X from 'lucide-react/dist/esm/icons/x';
 import ChevronDown from 'lucide-react/dist/esm/icons/chevron-down';
 import BarChart3 from 'lucide-react/dist/esm/icons/bar-chart-3';
 import RefreshCw from 'lucide-react/dist/esm/icons/refresh-cw';
-import { COLORS } from '../../components/common/UIComponents.jsx';
+import { useToast } from '../../components/common/UIComponents.jsx';
```

ESLint

[error] 'useToast' is defined but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)